release_notes/41-release-notes.adoc

```diff
@@ -0,0 +1,438 @@
+=== eBPF collection method on {ibm-z} and {ibm-linuxone}
+
+With this release, {product-title-short} has extended support for the eBPF collection method to {ibm-z} and {ibm-linuxone}. Now you can use this method to capture and analyze process and network activity in your OpenShift cluster on {ibm-z} and {ibm-linuxone}.
+
+[id="ability-to-configure-the-display-of-default-compliance-standards-in-the-compliance-dashboard_{context}"]
+=== Ability to configure the display of default compliance standards in the Compliance Dashboard
+
+You can now simplify and focus the Compliance Dashboard results by selecting only the built-in default standards you want to see in your results. You can now configure the dashboard to display default compliance standards, such as CIS Docker v1.2.0 and HIPAA 164.
+
+You can click the *Manage standards* button in the Compliance Dashboard to remove the standards that you do not want to display in the dashboard. For more information, see xref:../operating/manage-compliance.adoc#run-compliance-scan_manage-compliance[Running a compliance scan].
+
+[NOTE]
+====
+Standards controlled by the compliance operator cannot be disabled or hidden in {product-title-short}. You must configure the compliance operator custom resource definitions (CRDs) to remove them from display.
+====
+
+[id="declarative-configurations-for-authentication-and-authorization_{context}"]
+=== Declarative configurations for authentication and authorization
+
+Customers following the cloud native paradigm, prefer to use declarative configurations for versioning and auditability. Declarative configurations allow you to use configuration management tools to configure and update resources in your environment, rather than configuring resources directly in the system each time you want to make changes.
+
+After using declarative configuration to specify parameters for resources, you can quickly update configurations that are stored in repositories and apply them to the system using a GitOps workflow.
+
+You can create declarative configurations for authentication and authorization resources such as authentication providers, roles, permission sets, and access scopes. Configurations are stored in YAML files and mounted during Central installation.
+For more information, see xref:../operating/manage-user-access/manage-role-based-access-control-3630.adoc#declarative-configuration-auth-resources_manage-role-based-access-control[Declarative configuration for authentication and authorization resources].
+
+[id="sso-configuration-using-roxctl-cli_{context}"]
+=== SSO configuration using `roxctl` CLI
+
+In {product-title-short} 4.1, you can enhance Central's security and login capabilities with the new feature of configuring an authentication provider with `roxctl` CLI.
+This feature enables you to use the `roxctl` CLI and integrate with authentication systems such as OpenID Connect (OIDC) for a seamless single sign-on (SSO) experience.
```

Comment:
@dhaus67 Please can you suggest a more appropriate doc text, thx!

Reply:
```suggestion
In {product-title-short} 4.1, you can enhance Central's security and login capabilities with the new feature of using an authentication provider to authorize `roxctl` CLI.
This feature enables you to use the `roxctl` CLI and integrate with authentication systems such as OpenID Connect (OIDC) for a seamless single sign-on (SSO) experience without requiring additional tokens or passwords.
```